tools/gcc-bridge/compiler/src/main/java/org/renjin/gcc/Gcc.java

```diff
@@ -121,9 +122,12 @@ public GimpleCompilationUnit compileToGimple(File source, String... compilerFlag
    * @throws IOException
    * @throws GccException if the GCC process does not exit successfully
    */
-  private String callGcc(List<String> arguments) throws IOException {
+  private String callGcc(List<String> arguments, String extension) throws IOException {
     List<String> command = Lists.newArrayList();
-    command.add("gcc-4.6");
+    if("cpp".equals(extension))
```

Comment:
Excellent! Would love to see gcc-bridge find use outside of renjin so happy to help in any way to get this working for other libraries.